plugins/hls-eval-plugin/test/info-util/InfoUtil.hs

```diff
@@ -0,0 +1,20 @@
+module InfoUtil
+  ( Eq
+  , Ord
+  , Foo (..)
+  , Bar (..)
+  , Baz
+  )
+where
+
+import Prelude (Eq, Ord)
+
+data Foo = Foo1 | Foo2
+  deriving (Eq, Ord)
```

Comment:
I ended up dropping the `Show` instances since they would still come out in different orders sometimes. Thankfully `Eq`, `Ord` and `Baz` are enough to show the difference between `:info` and `:info!`